src/node_file.h

```diff
@@ -114,6 +114,8 @@ class FSReqBase : public ReqWrap<uv_fs_t> {
     return static_cast<FSReqBase*>(ReqWrap::from_req(req));
   }
 
+  FSReqBase(const FSReqBase&)=delete;FSReqBase&operator=(const FSReqBase&)=delete;
```

Comment:
Can you put this on separate lines + add the usual whitespace? (Also in 2 other places below)

Reply:
Sorry about that, still have not get `clang-format` works on my windows.